test/_init_tests.js

```diff
@@ -390,17 +386,22 @@ function preventAsyncErrorThrows() {
   stubAsyncErrorThrows();
 }
 
+before(function() {
+  // This is a more robust version of `this.skip()`. See #17245.
+  this.skipTest = function() {
+    if (this._runnable.title != '"before all" hook') {
+      throw new Error('skipTest() can only be called from within before()');
+    }
+    this.test.parent.pending = true;
```

Comment:
What's this?

Reply:
Workaround for https://github.com/mochajs/mocha/issues/2683.